lib/ansible/modules/network/aci/aci_access_port_to_interface_policy_leaf_profile.py

```diff
@@ -274,6 +282,13 @@ def main():
         ),
         child_classes=['infraPortBlk', 'infraRsAccBaseGrp']
     )
+
+    POLICY_GROUP_TYPE_MAPPING = dict(
+        port_channel='uni/infra/funcprof/accbundle-{0}'.format(policy_group),
```

Comment:
We probably want to support fex here too ?

Reply:
@dagwieers  Alright, I will check how fex are handled and will add support.